CHANGELOG.md

```diff
@@ -62,6 +62,15 @@ New Features
   truth information.  cf. demos 9 and 10. (#301, #691)
 - Added methods calculateHLR, calculateMomentRadius, and calculateFWHM to both
   GSObject and Image. (#308)
+- Added LookupTable2D to facilitate quick interpolation of two-dimensional
+  tabular data. (#465)
+- Added AtmosphericPhaseScreen, OpticalPhaseScreen, and PhaseScreenList used
+  to generate PSFs via Fourier optics. (#549)
+- Added PhaseScreenPSF to transform PhaseScreens into GSObjects.  (#549)
+- Added Atmosphere function to conveniently assemble a multi-layer atmosphere
+  PhaseScreenList. (#549)
+- Rewrote OpticalPSF to operate under the PhaseScreen framework to enable
```

Comment:
> Maybe we should at least move the script from /devel/modules/phase_psf/ to /examples/?

I think that's a good idea.

And good catch on the magically accelerating screens.